### ISSUE TITLE
: Improve Model Performance

### ISSUE THREAD

**Issue by Bil115** [Bil115/Predicting_Insurance_Charges#1]
The current prediction model for insurance charges has an RMSE of 3000. Improve the model by:  
- Experimenting with feature scaling techniques.  
- Trying alternative regression models like Random Forest or Gradient Boosting.  
- Hyperparameter tuning to optimize performance.  

Please document any changes made in the README.md file.

**Resources:**  
- [Scikit-learn regression models](https://scikit-learn.org/stable/supervised_learning.html#supervised-learning)  
- [Feature scaling techniques](https://scikit-learn.org/stable/modules/preprocessing.html#preprocessing)

**Comment by Bil115** [comment]
I will start by testing feature scaling methods (e.g., StandardScaler and MinMaxScaler) to see if they improve the model's performance. Results will be updated here soon.

**Comment by Bil115** [comment]
> I will start by testing feature scaling methods (e.g., StandardScaler and MinMaxScaler) to see if they improve the model's performance. Results will be updated here soon.



> I suggest trying Polynomial Regression for non-linear relationships.  
That's a great idea! I'll include it in the model experiments.

**Comment by Bil115** [comment]
> I will start by testing feature scaling methods (e.g., StandardScaler and MinMaxScaler) to see if they improve the model's performance. Results will be updated here soon.

> I suggest trying Polynomial Regression for non-linear relationships.  
That's a great idea! I'll include it in the model experiments.
